packages/react-components/react-breadcrumb/src/components/BreadcrumbDivider/BreadcrumbDivider.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { useBreadcrumbDivider_unstable } from './useBreadcrumbDivider';
+import { renderBreadcrumbDivider_unstable } from './renderBreadcrumbDivider';
+import { useBreadcrumbDividerStyles_unstable } from './useBreadcrumbDividerStyles';
+import type { BreadcrumbDividerProps } from './BreadcrumbDivider.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+
+/**
+ * BreadcrumbDivider component - TODO: add more docs
+ */
+export const BreadcrumbDivider: ForwardRefComponent<BreadcrumbDividerProps> = React.forwardRef((props, ref) => {
+  const state = useBreadcrumbDivider_unstable(props, ref);
+
+  useBreadcrumbDividerStyles_unstable(state);
```

Comment:
It would be a good idea to already include the custom styles hook, like in this PR:
https://github.com/microsoft/fluentui/pull/27078